pytrends/request.py

```diff
@@ -356,9 +358,6 @@ def related_queries(self):
         related_payload = dict()
         result_dict = dict()
         for request_json in self.related_queries_widget_list:
-            # ensure we know which keyword we are looking at rather than relying on order
```

Comment:
This functionality doesn't seem to be replaced.

Reply:
Hi, you're right I was so focused on getting the result without keywords I didn't check the alternatives. Now without keywords is just optional, when there are keywords nothing changes:

[https://github.com/tpilkati/pytrends/commit/6ca4733c1a23e45b6cfc084381466dcbd89e3d2b](url)

` if len(self.kw_list) > 0:`
`     # ensure we know which keyword we are looking at rather than relying on order`
`     kw = request_json['request']['restriction']['complexKeywordsRestriction']['keyword'][0]['value'] `              
`     result_dict[kw] = {'rising': df_rising, 'top': df_top}`
` else:`
`     result_dict = {'rising': df_rising, 'top': df_top}`